run/SU2_preCICE_CHT.py

```diff
@@ -0,0 +1,283 @@
+#!/usr/bin/env python3
+
+## File edited to work coupled with preCICE - Joseph Signorelli
+
+## \file launch_unsteady_CHT_FlatPlate.py
+#  \brief Python script to launch SU2_CFD with customized unsteady boundary conditions using the Python wrapper.
+#  \author David Thomas
+#  \version 7.4.0 "Blackbird"
```

Comment:
Seems to be an older copy?

Reply:
I think this is just the starting file attribution, I did not want to change that without deeply comparing the original files between versions.

@j-signorelli do you think we should change this comment to a newer version, or even just remove it? Do you know if SU2 7.5.0 is supported, or should we mention only 7.5.1 everywhere?

I can figure these all out by myself, but I would prefer a simple affirmation in case you already know.

Reply:
@MakisH that's just remnants of the original file I used as a starting point for implementation -- it's probably completely fine to just remove it altogether. It has been quite awhile so I don't remember but I do think SU2 v7.5.0 should also work just fine for this.

Reply:
Updated the headers in https://github.com/precice/su2-adapter/pull/40
I only need an approval in that, and will merge the release PR after that.